packages/kbn-securitysolution-io-ts-list-types/src/common/expire_time/index.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import * as t from 'io-ts';
+
+export const expireTime = t.string;
```

Comment:
Can't we use an ISO date type instead of the string here?

Reply:
I think this might work.

```suggestion
import * as t from 'io-ts';

import { IsoDateString } from '@kbn/securitysolution-io-ts-list-types';

export const expireTime = t.string;
```